src/components/user/user.service.ts

```diff
@@ -471,8 +465,8 @@ export class UserService {
     const response: any = {
       ...result,
       roles: {
-        value: result.roles.value || [],
-        canRead: result.roles.canEdit ?? result.roles.canRead,
+        value: result.roles.value,
+        canRead: result.roles.canEdit || result.roles.canRead,
```

Comment:
I believe this be done in the cypher mapping so we don't have to map here.

Reply:
![image](https://user-images.githubusercontent.com/4212008/89339078-e9197700-d6e0-11ea-8155-052c1ec56764.png)
If we don't set `roles` while registering user, the `canRead` will be `false` but that should be `true` if the `canEdit` is `true`

Reply:
Well that sounds like a different problem in general. Why would one be set but not the other?

Also doesn't this work?
```js
canRead: coalesce(canEdit, canRead, false)
```

Reply:
Yes, that's working. Thx